slangpy/benchmarks/conftest.py

```diff
@@ -1,3 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+# Importing these here means we don't have to explicitly import them into every benchmark
```

Comment:
True, but you still import the typings so IMO importing both is probably cleaner, but not fussy

Reply:
Hm, this avoids having to do the `# type: ignore` workaround so might still be worth it.